test/python/common.py

```diff
@@ -53,9 +53,8 @@ def with_ngraph(self, l, config=None):
         # because mutable objects should not be used as defaults in python
         if config is None:
             config = tf.compat.v1.ConfigProto()
-        # TODO: Stop grappler on failure (Add fail_on_optimizer_errors=True)
-        config = ngraph_bridge.update_config(config)
 
+        # config = ngraph_bridge.update_config(config)
```

Comment:
Disabling the config update; tests don't run with the optimizer?

Reply:
See below.